plugin/storage/es/spanstore/dbmodel/model.go

```diff
@@ -0,0 +1,77 @@
+package dbmodel
+
+import (
+	"github.com/uber/jaeger/model"
+)
+
+
+// ReferenceType is the reference type of one span to another
+type ReferenceType string
+
+// TraceID is a trace id found in a span
+type TraceID string
+
+// SpanID is the id of a span
+type SpanID uint64
+
+const (
+	// ChildOf means a span is the child of another span
+	ChildOf ReferenceType = "CHILD_OF"
+	// FollowsFrom means a span follows from another span
+	FollowsFrom ReferenceType = "FOLLOWS_FROM"
+)
+
+// Trace is a list of spans
+type Trace struct {
+	Spans []Span `json:"spans"`
+}
+
+// Span is a span denoting a piece of work in some infrastructure
+type Span struct {
```

Comment:
please keep field order the same as in the domain model https://github.com/uber/jaeger/blob/master/model/span.go#L54